src/library_browser.js

```diff
@@ -770,13 +770,10 @@ var LibraryBrowser = {
   emscripten_async_load_script__deps: ['$UTF8ToString'],
   emscripten_async_load_script: (url, onload, onerror) => {
     url = UTF8ToString(url);
-    onload = {{{ makeDynCall('v', 'onload') }}};
-    onerror = {{{ makeDynCall('v', 'onerror') }}};
-
 #if PTHREADS
     if (ENVIRONMENT_IS_PTHREAD) {
       err(`emscripten_async_load_script("${url}") failed, emscripten_async_load_script is currently not available in pthreads!`);
-      return onerror ? onerror() : undefined;
+      return onerror ? {{{ makeDynCall('v', 'onerror') }}}() : undefined;
```

Comment:
Is this change related?

Reply:
Yes, this is part of the change.  It makes it possible to safely call `exit` from one of the `emscripten_async_load_script` callbacks by wrapping them in `callUserCallback` (rather then directly calling the).

Reply:
How does this make that safe? This change looks NFC to me, like a cleanup - `onerror` is just moved around. What am I missing? Or is this part NFC but the code later is what makes a difference?

Reply:
When the callbacks are called asyncronously below they are now called via callUserCallback.  Before they were not.